### DISCUSSION TITLE
How to support the Chinese character ?

### DISCUSSION THREAD

**Discussion by hcucumber** [jlesage/docker-video-duplicate-finder#11] [Q&A]
My folder's name is Chinese character. And the VDF can't display Chinese character. after run the docker, I install the glibc and other software for zh-cn suport . But it always cant display Chinese character.

**Comment by jlesage** [comment]
Did you set the `ENABLE_CJK_FONT` environment variable ?
https://github.com/jlesage/docker-video-duplicate-finder?tab=readme-ov-file#environment-variables

**Comment by Yxini-p** [comment]
![图片](https://github.com/user-attachments/assets/52683940-164d-4a76-944f-6801b61652e8)
![图片](https://github.com/user-attachments/assets/d3f6d413-9a33-4981-97d0-29cbec01ff37)
Yes, I have set it up; But it doesn't seem to be effective。
![图片](https://github.com/user-attachments/assets/e1952c94-b692-4094-8e2a-48e14f5b9138)
And after restarting the Docker container, an error occurred; But due to my low level of proficiency, I am unable to understand and resolve this error;
If you see it, please answer it, thank you;
